src/sdk-reference/cpp/1/server/get-config/snippets/get-config.cpp

```diff
@@ -0,0 +1,7 @@
+try {
+  kuzzle->server->getConfig();
+
+  std::cout << "Success" << std::endl;
```

Comment:
Can't we test something about the response of the method, possibly using some regular expressions?